packages/nextjs/app/builders/0xa63f3deC58506f43B721025ae9cBAd9F4F7C658a/page.tsx

```diff
@@ -0,0 +1,65 @@
+import { NextPage } from "next";
+import { GithubIcon } from "~~/components/assets/GithubIcon";
+
+const VhforbesProfilePage: NextPage = () => {
+  return (
+    <div className="flex flex-col md:flex-row justify-center mt-10">
+      <div className="flex flex-col items-center">
+        <div className="avatar left-2">
+          <div className="w-32 mr-4 mask mask-hexagon text-center">
+            <img src="https://avatars.githubusercontent.com/u/39014716?v=4" />
+          </div>
+        </div>
+
+        <a href="https://github.com/vhforbes" target="_blank">
+          <GithubIcon className="w-12 h-12 pb-1 fill-white mt-4 hover:fill-accent" />
```

Comment:
The Github icon is almost invisible on light mode haha. You could use the `dark:` util in tailwind to have a different fill  depending on mode (maybe something like `fill-black dark:fill-white`??)

![image](https://github.com/BuidlGuidl/batch4.buidlguidl.com/assets/2486142/85b8d93c-db24-4f2e-809d-9c2148f82c4c)

Reply:
Update: the "dark:" selector might not work. But you could use something like:

```
const { resolvedTheme } = useTheme();
const isDarkMode = resolvedTheme === "dark";
```

nothing important, just wanted to nitpick <3

Reply:
thanks for the tip! It would not work haha

Reply:
Check this tho :P

https://github.com/scaffold-eth/scaffold-eth-2/pull/810